tests/datasets/test_benin_cashews.py

```diff
@@ -49,9 +50,45 @@ def dataset(
         )
         root = str(tmp_path)
         transforms = nn.Identity()  # type: ignore[attr-defined]
+        bands = BeninSmallHolderCashews.ALL_BANDS
+
         return BeninSmallHolderCashews(
             root,
             transforms=transforms,
+            bands=bands,
+            download=True,
+            api_key="",
+            checksum=True,
+            verbose=True,
+        )
+
+    @pytest.fixture()
+    def single_band_dataset(
```

Comment:
This duplicates most of the code from the other fixture. Sorry, I don't think I explained this very well. What I'm suggesting is to reuse the same fixture in both plotting tests, but instead of plotting the dataset directly in `test_failed_plot`, create a new dataset with the same values as the valid dataset but change the bands. See #312 where I explain this better and it has now been implemented.

Reply:
ah nice, this is much better